orbit/pkg/update/runner.go

```diff
@@ -159,6 +176,12 @@ func (r *Runner) UpdateAction() (bool, error) {
 			}
 			log.Info().Str("target", target).Msg("update completed")
 			didUpdate = true
+			if target == "nudge" {
+				r.mu.Lock()
+				r.localHashes["nudge"] = metaHash
+				r.mu.Unlock()
+				log.Info().Msgf("updating local hash(%s)=%x", "nudge", r.localHashes["nudge"])
+			}
```

Comment:
(done in linked PR) I don't think we should do this here, instead we should recompute the hash outside of this function.